components/l10n/common/locale_util.cc

```diff
@@ -5,78 +5,113 @@
 
 #include "brave/components/l10n/common/locale_util.h"
 
-#include <algorithm>
-#include <vector>
-
-#include "base/strings/string_split.h"
-#include "base/strings/string_util.h"
-#include "base/strings/utf_string_conversions.h"
-#include "ui/base/resource/resource_bundle.h"
+#include "absl/types/optional.h"
+#include "base/check.h"
+#include "base/no_destructor.h"
+#include "brave/components/l10n/common/locale_constants.h"
+#include "third_party/icu/source/common/unicode/locid.h"
+#include "ui/base/l10n/l10n_util.h"
 
 namespace brave_l10n {
 
 namespace {
 
-const char kDefaultLanguageCode[] = "en";
-const char kDefaultCountryCode[] = "US";
+inline icu::Locale GetNormalizedICULocale(const std::string& locale) {
+  return {l10n_util::NormalizeLocale(locale).c_str()};
+}
 
-}  // namespace
+inline std::string GetLocale(const icu::Locale& icu_locale) {
+  std::string locale = {icu_locale.getBaseName()};
```

Comment:
We shouldn't need the curly braces here.

Reply:
Removed, thanks

Reply:
Fixed :-)